modules/appeals_api/spec/docs/higher_level_reviews/v0_spec.rb

```diff
@@ -106,7 +107,7 @@ def swagger_doc
     end
   end
 
-  path '/forms/200996/{uuid}' do
+  path '/forms/200996/{id}' do
```

Comment:
I changed this parameter to be called `id` instead of `uuid` so that it matches the other `show` endpoints in the segmented APIs - this isn't directly related to the other changes in this PR